spec/models/user_spec.rb

```diff
@@ -20,14 +20,11 @@
 #  confirmation_sent_at   :datetime
 #  unconfirmed_email      :string
 #  name                   :string
-#  nickname               :string
```

Comment:
Is this right?

Reply:
Yes, this is an annotation from a [previous PR](https://github.com/sebastiancaraballo/target/pull/28).